pkg/scalers/azure_pipelines_scaler.go

```diff
@@ -139,7 +139,7 @@ func (s *azurePipelinesScaler) GetAzurePipelinesQueueLength(ctx context.Context)
 	r.Body.Close()
 
 	if !(r.StatusCode >= 200 && r.StatusCode <= 299) {
-		return -1, fmt.Errorf("azure Devops REST api returned error. status: %d response: %s", r.StatusCode, string(b))
+		return -1, fmt.Errorf("Azure Devops REST api returned error. url: %s status: %d response: %s", string(url, r.StatusCode, string(b))
```

Comment:
[The official name is Azure DevOps AFAIK](https://azure.microsoft.com/en-us/solutions/devops/)
```suggestion
		return -1, fmt.Errorf("Azure DevOps REST api returned error. url: %s status: %d response: %s", string(url, r.StatusCode, string(b))
```